src/server/db/schema/findingAttachment.ts

```diff
@@ -52,6 +52,7 @@ export const insertFindingAttachmentSchema = createInsertSchema(
   {
     attachmentUrl: (schema) =>
       schema.attachmentUrl.url('Invalid attachment URL.'),
+    mimeType: (schema) => schema.mimeType.min(1, 'MIME type can’t be empty.'),
```

Comment:
Actually `mimeType` can be optional as the file type may not be always recognized by vercel, but I have it changed in my upcoming PR.